integration/eth2network/eth2_network.go

```diff
@@ -556,3 +559,10 @@ func (n *Impl) gethImportEnodes(enodes []string) error {
 	}
 	return nil
 }
+
+func min(a, b int) int {
```

Comment:
I'd guess there should be a `math.Min` function

Reply:
W have generics now. Maybe it works

Reply:
They haven't added them to the core library yet afaik. But I'll see if there's a third party lib or at least just put it into our own util next time I'm at desk

Reply:
No worries. It’s just a curiosity. Seems like a pretty common thing, but with golang you never know

Reply:
Yeah I think they haven't worked it out yet. Because math.Min/Max is for floats only I guess they don't want to change compatibility. I'll leave it for now, but yeah we should keep an eye out for best practice on the sorts of utils generics unlock.